### PR TITLE
treewide: remove vrthra, cpages and lovek323 from meta.maintainers

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

Since they are not active anymore.

https://github.com/NixOS/nixpkgs/pull/310759

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
@vrthra gentle ping

File: pkgs/development/interpreters/guile/2.0.nix [hunk 1]
```diff
@@ -152,7 +152,7 @@ builder rec {
         processing.
       '';
     license = licenses.lgpl3Plus;
-    maintainers = with maintainers; [ ludo lovek323 vrthra ];
```
Comment (thiagokokada) [inline pkgs/development/interpreters/guile/2.0.nix:155]: `lovek323` removed by mistake?

File: pkgs/development/interpreters/guile/2.2.nix [hunk 2]
```diff
@@ -142,7 +142,7 @@ builder rec {
       foreign function call interface, and powerful string processing.
     '';
     license = licenses.lgpl3Plus;
-    maintainers = with maintainers; [ ludo lovek323 vrthra ];
```
Comment (thiagokokada) [inline pkgs/development/interpreters/guile/2.2.nix:145]: Here too.

File: pkgs/development/libraries/tk/generic.nix [hunk 3]
```diff
@@ -69,6 +69,6 @@ tcl.mkTclDerivation {
     homepage = "https://www.tcl.tk/";
     license = licenses.tcltk;
     platforms = platforms.all;
-    maintainers = with maintainers; [ lovek323 vrthra ];
```
Comment (thiagokokada) [inline pkgs/development/libraries/tk/generic.nix:72]: Here too.

**Comment by bjornfor** [comment]
> treewide: remove vrthra from meta.maintainers 

Please explain in the commit message why.

**Review by thiagokokada — APPROVED** [review]

**Comment by AndersonTorres** [comment]
> > treewide: remove vrthra from meta.maintainers
> 
> Please explain in the commit message why.

Done

**Review by superherointj — APPROVED** [review]
If they ever want to come back, they can always re-add themselves.

**Comment by AndersonTorres** [comment]
P.S.: some packages still refer to them. I  am reworking these packages in other PRs, some of them into Staging.

**Pull request merged**